app/main/routes_frontend.py

```diff
@@ -418,40 +418,44 @@ def setup():
                 # sudo sed -i -e"s/^psk=.*/psk=\"$WIFIPASS\"/" /etc/wpa_supplicant/wpa_supplicant.conf
-            
+
                 try:
-                    wpa_files = "/etc/wpa_supplicant/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant-wlan0.conf"
+                    # <= beta4 => /etc/wpa_supplicant/wpa_supplicant.conf
+                    # >= beta5 => /etc/wpa_supplicant/wpa_supplicant-wlan0.conf
+                    wpa_files = " ".join([_x for _x in ("/etc/wpa_supplicant/wpa_supplicant.conf", "/etc/wpa_supplicant/wpa_supplicant-wlan0.conf") if os.path.exists(_x)])
 
                     # set ssid in wpa_supplicant files
+                    # regex \b marks a word boundary
                     ssid = payload['ssid']
                     subprocess.check_output(
-                        """sed -i -e 's/ssid=.*/ssid="{}"/' {}""".format(ssid, wpa_files), shell=True)
-                    
+                        """sed -i 's/\bssid=.*/ssid="{}"/' {}""".format(ssid, wpa_files), shell=True)
+
                     # set bssid (if set by user) in wpa_supplicant files
                     if 'bssid' in payload and payload['bssid']:
                         bssid = payload['bssid']
                         # remove comment for bssid line (if present)
                         subprocess.check_output(
                             """sudo sed -i '/bssid/s/# *//g' {}""".format(wpa_files), shell=True)
                         subprocess.check_output(
-                            """sed -i -e 's/bssid=.*/bssid={}/' {}""".format(bssid, wpa_files), shell=True)
+                            """sudo sed -i 's/bssid=.*/bssid={}/' {}""".format(bssid, wpa_files), shell=True)
                     else:
-                        # add comment for bssid line (if present)
+                        # add a comment for bssid line (if present)
                         subprocess.check_output(
-                            """sudo sed -i '/bssid/s/bssid/# bssid/g' {}""".format(wpa_files), shell=True)
-                    
+                            """sudo sed -i 's/\(bssid=.*\)/# \\1/g' {}""".format(wpa_files), shell=True)
+
                     # set credentials (if set by user) in wpa_supplicant files
                     if 'password' in payload and payload['password']:
                         psk = payload['password']
                         subprocess.check_output(
-                            """sed -i -e 's/psk=.*/psk="{}"/' {}""".format(psk, wpa_files), shell=True)
-                    
+                            """sed -i 's/psk=.*/psk="{}"/' {}""".format(psk, wpa_files), shell=True)
+
                     def restart_wireless():
                         import subprocess
                         import time
                         time.sleep(2)
+                        current_app.logger.info("applying changes and restarting wireless interface")
```

Comment:
This is something I added after testing... needs to be moved outside of the `restart_wireless` subprocess method... as there is no known application context here.